data/mods/More_Survival_Tools/snippets.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "type": "snippet",
+    "category": "note",
+    "//": "Random graveyard epitaph from Nethack, and also referencing prehistoric hunter profession in the mod.",
```

Comment:
Not sure how to attribute it. Wiki only says it "might" be an Internet Oracle reference, and further cites how the caveman class can freely commit cannibalism.